.github/workflows/lint.yml

```diff
@@ -1,6 +1,6 @@
 name: Run linters
 
-on: [push]
+on: [push, workflow_dispatch]
```

Comment:
Shouldn't this be as below for consistency with other projects?
```
on:
  push:
    branches:
      - main
  pull_request:
```

Reply:
Yeah, you're right, changed in d451cc4.